### PR TITLE
Add debuggees and projects description in dropdown list

### PR DESCRIPTION
**Pull request by hananeikhelef**
**Work Done**

- Add debuggees description in the drop-down list 

**Authors** 
- @jamohile  & @hananeikhelef

### CODE REVIEW

File: app/src/client/chathead/Chathead.tsx [hunk 1]
```diff
@@ -31,6 +31,8 @@ interface ChatheadProps {
   activeBreakpoints: BreakpointMeta[];
   /** Breakpoints that have been hit, with all data loaded. */
   completedBreakpoints: Breakpoint[];
+  /** The current selected project descripttion. Undefined if no project selected. */
+  projectDescription: string | undefined;
```
Comment (jamohile) [inline app/src/client/chathead/Chathead.tsx:35]: I don't believe this is being used anywhere, can be removed

File: app/src/client/chathead/SelectProject.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { BackgroundRequestError } from "../../common/requests/BackgroundRequest"
 
 interface SelectProjectContainerProps {
   projectId?: string;
+  projectDescription?: string;
```
Comment (jamohile) [inline app/src/client/chathead/SelectProject.tsx:11]: Unused

File: app/src/client/injected/injected-app.tsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ import { BreakpointMarkers } from "../markers/BreakpointMarkers";
 interface InjectedAppState {
   projectId: string;
   debuggeeId: string;
+  projectDescription: string;
```
Comment (jamohile) [inline app/src/client/injected/injected-app.tsx:11]: Same

File: app/src/common/types/debugger.ts [hunk 4]
```diff
@@ -42,6 +42,7 @@ export interface Project {
     lifecycleState: string;
     name: string;
     createTime: string;
+    description: string;
```
Comment (jamohile) [inline app/src/common/types/debugger.ts:45]: Turns out this isn't available :(

File: app/src/client/chathead/Chathead.tsx (hunk 1 above)
Comment (hananeikhelef) [reply app/src/client/chathead/Chathead.tsx]: Done ^^

File: app/src/client/chathead/SelectProject.tsx (hunk 2 above)
Comment (hananeikhelef) [reply app/src/client/chathead/SelectProject.tsx]: Done ^^

File: app/src/client/injected/injected-app.tsx (hunk 3 above)
Comment (hananeikhelef) [reply app/src/client/injected/injected-app.tsx]: Done ^^

File: app/src/common/types/debugger.ts (hunk 4 above)
Comment (hananeikhelef) [reply app/src/common/types/debugger.ts]: deleted :)

File: app/src/client/chathead/GeneralSelectView.tsx [hunk 5]
```diff
@@ -6,8 +6,10 @@ interface SelectViewProps {
   label?: string;
   optionsLoading: boolean;
   selectedOptionId?: string;
+  projectDescription?: string;
```
[the remark this answers is not in the document]
Comment (hananeikhelef) [reply app/src/client/chathead/GeneralSelectView.tsx]: Done ^^

**Pull request merged**